Features/Stories/FieldTypes/IntegerFieldType.feature

```diff
@@ -0,0 +1,202 @@
+Feature: Test the validations done on fields from PlatformUI - Integer fieldtype
+    In order to validate the integer fieldtype
+    As an Editor  user
+    I need to be able to create, update and delete content with integer fieldtypes
+
+    #Background:
+     #   Given I am logged in as an Editor in PlatformUI
+
+        ##
+        # Validate the existence of expected fields from a field type when creating a content
+        ##
+    @javascript
+    Scenario: A Content of a Content Type that has an integer fieldtype must have an integer field
+        Given a Content Type with an "integer" Field exists
+        When I create a content of this type
+        Then I should see an "integer" field
+
+    @javascript
+    Scenario: When editing a Content the label of an integer field must have the same name than field type from the respective Content Type
+        Given a Content Type with an "integer" with field definition name "Quantity" exists
+        When I create a content of this type
+        Then I should see a "Quantity" label related with the "integer" field
+
+    @javascript
+    Scenario: The label of an mandatory integer field of a Content must be marked as mandatory
+        Given a Content Type with a required "integer" with field definition name "Quantity" exists
+        When I create a content of this type
+        Then the "Quantity" field should be marked as mandatory
+
+        ##
+        # Creating Content using a Content Type that has an Integer Field Type
+        ##
+    @javascript
+    Scenario: Publishing a valid integer Field works
```

Comment:
Publishing -> Creating, given you use Updating below that strictly also implies publishing